src/analysis/return_value.rs

```diff
@@ -63,6 +78,7 @@ pub fn analyze(env: &Env, func: &library::Function, type_tid: library::TypeId,
         parameter: parameter,
         base_tid: base_tid,
         commented: commented,
+        bool_return_is_error: if bool_return_is_error { Some(func.c_identifier.as_ref().unwrap().clone()) } else { None },
```

Comment:
Please, move it to variable. And IMHO "g_setenv" more confising "glib::setenv".
May be just change config value to string?